kubernetes/zenko/charts/backbeat/dashboards/backbeat.json

```diff
@@ -0,0 +1,543 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
+      "label": "Prometheus",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    }
+  ],
+  "__requires": [
+    {
+      "type": "grafana",
+      "id": "grafana",
+      "name": "Grafana",
+      "version": "5.1.3"
+    },
+    {
+      "type": "panel",
+      "id": "graph",
+      "name": "Graph",
+      "version": "5.0.0"
+    },
+    {
+      "type": "datasource",
+      "id": "prometheus",
+      "name": "Prometheus",
+      "version": "5.0.0"
+    }
+  ],
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": null,
+  "links": [],
+  "panels": [
+    {
+      "collapsed": false,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 6,
+      "panels": [],
+      "title": "CRR",
+      "type": "row"
+    },
+    {
+      "aliasColors": {
+        "crr_number_of_sites{app=\"backbeat-api\",chart=\"backbeat-1.0.0-RC5\",heritage=\"Tiller\",instance=\"10.1.14.56:8900\",job=\"kubernetes-service-endpoints\",kubernetes_name=\"zenko-backbeat-api\",kubernetes_namespace=\"default\",release=\"zenko\"}": "#806eb7"
```

Comment:
will `instance` ip vary here? Also `kubernetes_name` depending on deployment name?

Also, if possible, maybe use variables for things like `chart`

Reply:
These fields don't really matter. I will delete them.